setup.cfg

```diff
@@ -46,6 +46,7 @@ install_requires =
     asn1crypto>0.24.0,<2.0.0
     cffi>=1.9,<2.0.0
     cryptography>=3.1.0,<43.0.0
+    paramiko
```

Comment:
@sfc-gh-jfan We added a new dependency to python connector. Does it need to be reviewed by your team, or a tool has checked it?